swagger.yaml

```diff
@@ -2201,12 +2201,14 @@ definitions:
   host-role-update-params:
     type: string
     enum:
+      - 'auto-assign'
       - 'master'
       - 'worker'
 
   host-role:
     type: string
     enum:
+      - 'auto-assign'
```

Comment:
I think that Swagger should consist from entities that are the building blocks that build the API.  auto-assign is an implementation detail of how it was decided to develop the this feature.  It is not a valid host-role and should not be part of the Swagger.
BTW, Isn't empty role is better? Why is it needed?

Reply:
User can change role to auto-assign so why is it not a building block?

Reply:
auto-assign is not a real role.

Reply:
@ori-amizur What is your suggestion?  I don't see a problem here.  The user can set the role to "auto-assign", and when they look at the host they will see "auto-assign".  Sure, that role isn't valid once the host starts the installation, but I don't see a problem with it.  And it is definitely more clear and explicit than empty role.

Reply:
@avishayt I think empty role is a missing one.

Reply:
We don't have empty role anymore. 
The three options are auto-assign, worker, master
The default is auto-assign